liquibase.xml

```diff
@@ -0,0 +1,292 @@
+				value="org.openmrs.module.isantepluspatientdashboard.formHistory" />
+			<column name="enabled" value="1" />
+			<column name="component_type" value="EXTENSION" />
+			<column name="uuid" value="da7cf364-7110-11e6-8b77-86f30ca893d3" />
+		</insert>
+	</changeSet>
+	<changeSet id="2016Sept02-1526" author="k-joseph">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="0">
+				SELECT COUNT(*) FROM
+				appframework_component_state WHERE component_id =
+				'org.openmrs.module.isantepluspatientdashboard.labHistory';
+			</sqlCheck>
+		</preConditions>
+		<comment></comment>
+		<insert tableName="appframework_component_state">
+			<column name="component_id"
+				value="org.openmrs.module.isantepluspatientdashboard.labHistory" />
+			<column name="enabled" value="1" />
+			<column name="component_type" value="EXTENSION" />
+			<column name="uuid" value="da7cf710-7110-11e6-8b77-86f30ca893d3" />
+		</insert>
+	</changeSet>
+	<changeSet id="2016Sept02-1527" author="k-joseph">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="0">
+				SELECT COUNT(*) FROM
+				appframework_component_state WHERE component_id =
+				'org.openmrs.module.isantepluspatientdashboard.formsHistory';
+			</sqlCheck>
+		</preConditions>
+		<comment></comment>
+		<insert tableName="appframework_component_state">
+			<column name="component_id"
+				value="org.openmrs.module.isantepluspatientdashboard.formsHistory" />
+			<column name="enabled" value="1" />
+			<column name="component_type" value="EXTENSION" />
+			<column name="uuid" value="da7cf8dc-7110-11e6-8b77-86f30ca893d3" />
+		</insert>
+	</changeSet>
+	<changeSet id="2016Sept02-1528" author="k-joseph">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="0">
+				SELECT COUNT(*) FROM
+				appframework_component_state WHERE component_id =
+				'org.openmrs.module.isantepluspatientdashboard.growthCharts';
+			</sqlCheck>
+		</preConditions>
+		<comment></comment>
+		<insert tableName="appframework_component_state">
+			<column name="component_id"
+				value="org.openmrs.module.isantepluspatientdashboard.growthCharts" />
+			<column name="enabled" value="1" />
+			<column name="component_type" value="EXTENSION" />
+			<column name="uuid" value="3804126a-7111-11e6-8b77-86f30ca893d3" />
+		</insert>
+	</changeSet>
+	<changeSet id="2016Sept02-1529" author="k-joseph">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="0">
+				SELECT COUNT(*) FROM
+				appframework_component_state WHERE component_id =
+				'org.openmrs.module.isantepluspatientdashboard.patientDashboard.secondColumnFragments.weightGraph';
+			</sqlCheck>
+		</preConditions>
+		<comment></comment>
+		<insert tableName="appframework_component_state">
+			<column name="component_id"
+				value="org.openmrs.module.isantepluspatientdashboard.patientDashboard.secondColumnFragments.weightGraph" />
+			<column name="enabled" value="1" />
+			<column name="component_type" value="EXTENSION" />
```

Comment:
this file is in the wrong place, what happened to your repo to have these files misplaced